doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                Tim is using a version control system and can extract the code from the correct time period and reproduce the data.
+                <br/>
+                Alan did not use version control and just changes the code every time the rates changed.
+                <br/>
+                He has to look-up the correct values and rewrite the code before he can reproduce the data.
+                <br/>
+                <br/>
+                Scenario 4
+                <br/>
+                Marsha and Twist work in fashion marketing, Marsha has provided Twist with some market changes forecasting code from a version control repository.
+                <br/>
+                The code forecasts the supply and demand trend for the coming month based on previous years data and trends.
+                <br/>
+                Twist made some changes to her code in her local/working copy and ran the code.
+                <br/>
+                This produced some very unrealistic results and Twist realised that she had made some fundamental mistakes.
+                <br/>
+                As she had not committed her changes to the version control repository then she can just revert her local changes and start again.
+                <br/>
+                <br/>
+                Scenario 5
+                <br/>
+                Following on from scenario 4 Twist this time made the changes and the code failed but this time she had committed the code to the repository.
+                <br/>
+                It is all okay as she can go back to the last checked in version that she know worked as expected using version control.
+                <br/>
+                <br/>
+                Scenario 6
+                <br/>
+                A group of software engineers, Shaun, Ed, Liz, Dianne and David, are working on project to improve the scheduling and running of weather forecast models.
+                <br/>
+                They are all continuously getting the code from a version controlled repository and making changes, testing it and if it improve the system they commit it back to the trunk for others to then use.
+                <br/>
+                The others in the group can then "update" their local copy to the trunk version using the tools provided.
+                <br/>
+                <br/>
+                Scenario 7
+                <br/>
+                Following on from scenario 6 at a later date David decides that he wants to deviate from the trunk and is not sure it will work so decides to go it alone, but he may want to apply his change to the trunk at a later date.
+                <br/>
+                David can do this by branching from the trunk, making changes to his branch and testing them.
+                <br/>
+                David can then either delete the branch if the results are not improved or when he is happy that the changes are suitable he can merge his branch changes into the up to date trunk.
```

Comment:
Ordering of sentence is slightly confusing (to me) on first read as seems to suggest David can delete the branch if the results are not or improved or when he is happy with the change. Perhaps:

David can then either delete the branch if the results are not improved or he can merge his branch changes into the up to date trunk when he is happy that the changes are suitable.